packages/client/README.md

```diff
@@ -8,4 +8,37 @@ proteus-js-client allows Javascript apps to connect to a Proteus broker and acce
 yarn add proteus-js-client
 ```
 
+The Proteus client should be passed a configuration object. Here are its properties:
+
+```
+const config = {
+  setup: {
+    destination: // TODO: remove?
```

Comment:
I think this is kind of a migration thing or shortcut - in Java, I believe we will create a tag for this input if the tag is not present.

Reply:
I'll just leave it out of the readme then - thanks.